networkx/algorithms/bipartite/tests/test_matching.py

```diff
@@ -82,3 +84,13 @@ def test_to_vertex_cover(self):
         matching = maximum_matching(self.graph)
         vertex_cover = to_vertex_cover(self.graph, matching)
         self.check_vertex_cover(vertex_cover)
+
+
+def test_eppstein_matching():
+    # Special test
```

Comment:
Of course, someone has already had this problem and solved it a long time ago. In that case, my preference is leaving docstrings as docstrings for tests, and I will just install that plugin on my own machine and use it when I run `nosetests`.